app/scripts/controller-init/utils.ts

```diff
@@ -36,7 +36,11 @@ type ControllerMessengerCallback = (
   BaseControllerMessenger: BaseControllerMessenger,
 ) => BaseRestrictedControllerMessenger;
 
-type ControllersToInitialize = 'PPOMController' | 'TransactionController';
+type ControllersToInitialize =
+  | 'PPOMController'
+  | 'TransactionController'
+  | 'RateLimitController'
+  | 'SnapController';
```

Comment:
What about the CronjobController, SnapInterfaceController etc?

Reply:
Ah 👍

Reply:
Line 48 can probably be removed after this change.